catalog/spatial/kml/spatial-kml-transformer/src/test/java/org/codice/ddf/spatial/kml/transformer/KMLTransformerImplTest.java

```diff
@@ -222,6 +248,71 @@ public void testPerformDefaultTransformationMultiLineStringLocation()
     assertThat(multiLineString.getGeometry().get(1), instanceOf(LineString.class));
   }
 
+  @Test
+  public void testPerformDefaultTransformationExtendedData()
+      throws CatalogTransformerException, DateTimeParseException {
+    MetacardImpl metacard = createMockMetacard();
+
+    metacard.setLocation(POINT_WKT);
+    metacard.setCreatedDate(METACARD_DATE);
+    metacard.setEffectiveDate(METACARD_DATE);
+    metacard.setExpirationDate(METACARD_DATE);
+    metacard.setModifiedDate(METACARD_DATE);
+    metacard.setTags(METACARD_TAGS);
+
+    final Set<AttributeDescriptor> attributeDescriptors =
+        metacard.getMetacardType().getAttributeDescriptors();
+
+    Placemark placemark = kmlTransformer.performDefaultTransformation(metacard);
+
+    final List<Data> dataList = placemark.getExtendedData().getData();
+
+    int dataCount = 0;
+    for (AttributeDescriptor attributeDescriptor : attributeDescriptors) {
+      final String attributeName = attributeDescriptor.getName();
+      final Attribute attribute = metacard.getAttribute(attributeName);
+      if (attribute != null) {
+        dataCount++;
+      }
+    }
+
+    assertThat(dataList.size(), is(dataCount));
+
+    for (Data data : dataList) {
+      switch (data.getName()) {
+        case "id":
+          assertThat(data.getValue(), is(METACARD_ID));
+          break;
+        case "title":
+          assertThat(data.getValue(), is(METACARD_TITLE));
+          break;
+        case "location":
+          assertThat(data.getValue(), is(POINT_WKT));
+          break;
+        case "metadata-content-type":
+          assertThat(data.getValue(), is(METACARD_TYPE));
+          break;
+        case "metadata-content-type-version":
+          assertThat(data.getValue(), is(METACARD_VERSION));
+          break;
+        case "metadata":
+          assertThat(data.getValue(), is(METACARD_METADATA));
+          break;
+        case "metacard-tags":
+          assertThat(
+              data.getValue(),
+              is(METACARD_TAGS.asList().get(0) + ", " + METACARD_TAGS.asList().get(1)));
```

Comment:
I think if you made `METACARD_TAGS` a list then you can just `assertThat(data.getValue(), is(METACARD_TAGS)`